module/zfs/arc.c

```diff
@@ -7471,8 +7472,8 @@ arc_init(void)
 	cv_init(&arc_reclaim_waiters_cv, NULL, CV_DEFAULT, NULL);
 
 	/* Convert seconds to clock ticks */
```

Comment:
`s/seconds/milliseconds/` and it would be good to drop all mention of clock ticks here since this isn't relevant except at the place in the code where the value is actually used.